### PR TITLE
fixing dataview selection

### PR DESCRIPTION
**Pull request by hbaptiste**
fixing dataview selection

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
:+1:

File: src/tb/component/medialibrary/main.js [hunk 1]
```diff
@@ -60,8 +60,8 @@ define(
                     width: jQuery(window).width() - (20 * 2)
                 },
                 rangeSelector: {
-                    range: [1, 50, 1],
-                    selected: 1//10
+                    range: [10, 50, 10],
```
Comment (ndufreche) [inline src/tb/component/medialibrary/main.js]: you forgot a 0 I think, `[10, 50, 10]` => `[10, 50, 100]`

Comment (hbaptiste) [inline src/tb/component/medialibrary/main.js]: it's a step. Step 10 from 10 to 50

Comment (ndufreche) [inline src/tb/component/medialibrary/main.js]: Ho ok :)

**Comment by ndufreche** [comment]
:+1: thanks

**Pull request merged**